atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraClientPool.java

```diff
@@ -207,7 +205,7 @@ private boolean isHostHealthy(InetSocketAddress host) {
         try {
             CassandraClientPoolingContainer testingContainer = new CassandraClientPoolingContainer(host, config);
             testingContainer.runWithPooledResource(describeRing);
-            testingContainer.runWithPooledResource(CassandraVerifier.healthCheck);
+            testingContainer.runWithPooledResource(validatePartitioner);
```

Comment:
and yes, healthCheck is still used, it has its place in the code I wrote in the last PR to determine if hosts are so badly screwy during the bootup process that we shouldn't even try doing anything else with them.